modules/local/hifiadapterfilt/main.nf

```diff
@@ -0,0 +1,55 @@
+process HIFIADAPTERFILT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container 'quay.io/microbiome-informatics/hifiadapterfilt:v1.0.1'
+
+    containerOptions {
+        if (workflow.containerEngine == 'singularity') {
+            return "--bind ${moduleDir}/tests/fixtures/:/data/"
+        } else {
+            return "-v ${moduleDir}/tests/fixtures/:/data/"
+        }
+    }
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta), path("*.filt.fastq.gz"), emit: filt
+    path("*.contaminant.blastout")          , emit: blast_search
+    path("*.stats")                         , emit: stats
+    path("*.blocklist")                     , emit: headers
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    
+    """
+    hifiadapterfilt.sh ${args}
```

Comment:
I guess this one will be replaced with the nf-core one

Reply:
Correct, hence the temporary incomplete status